src/feed/record/MultiFormatRecord.ts

```diff
@@ -30,7 +30,7 @@ export class MultiFormatRecord implements Record {
       values[key] = record[key].extract(line.substr(record[key].position, record[key].length));
     }
 
-    return { action, values };
+    return { action, values, keysValues: values };
```

Comment:
Shouldn't this get just they keys' values?

Reply:
it's hardcoded to RecordAction.Insert so it doesn't matter but maybe for consistency I should tidy it up

Reply:
Maybe put an empty object and a comment here.